service-map.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "Service",
+    "name": "psc-discrepancy-parser",
+    "description": "This AWS Lambda service processes batches of CSV-format survey results (each survey gathered discrepancies with a company's Person(s) of Significant Control (PSC)), transforming a survey (in the form of a CSV record) into JSON and sending it to CHIPS.",
+    "owned_by": "Titans",
+    "git_repo": "https://github.com/companieshouse/psc-discrepancy-parser",        
+    "users":"Obligied Entities",    
```

Comment:
This is an internal Lambda - maybe this should be clearer that it is used internally to process public-supplied data